src/components/TechStackSection/TechStackSection.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from 'styled-components';
+import { rem } from 'polished';
+import {
+  DiCss3,
+  DiGit,
+  DiGithub,
+  DiHtml5,
+  DiJavascript,
+  DiReact,
+} from 'react-icons/di';
+
+import {
+  SectionWrapper,
+  SectionHeader,
+  SectionItemWrapper,
+  Icon,
+  IconWrapper,
+} from 'components/Section/styled';
+import { breakPoints } from 'styles/base';
+
+const TechStackIconWrapper = styled(IconWrapper)`
+  width: 50%;
+  padding: ${rem(10)} ${rem(40)};
+
+  @media (min-width: ${breakPoints.desktop}) {
+    width: 30%;
+    padding: ${rem(20)} ${rem(20)};
+  }
+`;
+
+const TechStackSection = () => (
+  <SectionWrapper justifyContent="flex-start">
```

Comment:
I don't like this approach - you will kill yourself with tons of props with something like that. 
We can use built-in styled component mechanism. 
styled(SectionWrapper); 
Read errors, because you will have to handle classNames -> your task ;)